### PR TITLE
Change SDK imports to servicemeshinterface

### PR DESCRIPTION
**Pull request by stefanprodan**
Changes:
- update imports and go modules
- regenerate client set
- update docs to reflect the org change

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.13
-    working_directory: /go/src/github.com/deislabs/smi-sdk-go
+      - image: circleci/golang:1.14
```
Comment (lachie83) [inline .circleci/config.yml]: Can we break the go change out into a different PR please? I just don't want to conflate things.

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,6 @@
-module github.com/deislabs/smi-sdk-go
+module github.com/servicemeshinterface/smi-sdk-go
 
-go 1.13
+go 1.14
```
Comment (lachie83) [inline go.mod]: Separate PR please

**Comment by stefanprodan** [comment]
@lachie83 please take a look now
